saga-demo/saga-spring-cloud-demo/pom.xml

```diff
@@ -24,14 +24,14 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>booking</artifactId>
+  <artifactId>saga-spring-cloud-demo</artifactId>
   <name>Saga::Demo::Spring-Cloud-Demo</name>
   <packaging>pom</packaging>
 
   <modules>
-    <module>booking</module>
-    <module>car</module>
-    <module>hotel</module>
+    <module>pack-booking</module>
```

Comment:
It's better to name it to spring-cloud-booking.

Reply:
updated:

[INFO] Saga::Demo
[INFO] Saga::Demo::Dubbo-Demo
[INFO] Saga::Demo::Dubbo-Demo::ServicePub
[INFO] Saga::Demo::Dubbo-Demo::ServiceAPI
[INFO] Saga::Demo::Dubbo-Demo::ServiceA
[INFO] Saga::Demo::Dubbo-Demo::ServiceB
[INFO] Saga::Demo::Dubbo-Demo::ServiceC
[INFO] Saga::Demo::Spring-Cloud-Demo
[INFO] Saga::Demo::Spring-Cloud-Demo::Booking
[INFO] Saga::Demo::Spring-Cloud-Demo::Car
[INFO] Saga::Demo::Spring-Cloud-Demo::Hotel